shared/src/libs/enums/exception-message.enum.ts

```diff
@@ -1,11 +1,19 @@
+import { FileUploadValidationRule } from '#packages/files/files.js';
+
+import { ExceptionRule } from './exception-rule.enum.js';
+
+const BYTES_IN_MEGABYTE = 1_000_000;
```

Comment:
Create helper for calculation separately

Reply:
Done